src/Views/ShortcutsView.vala

```diff
@@ -26,6 +26,8 @@ public class ShortcutOverlay.ShortcutsView : Gtk.Grid {
     private const string SCHEMA_MEDIA = "org.gnome.settings-daemon.plugins.media-keys";
     private const string SCHEMA_MUTTER = "org.gnome.mutter.keybindings";
 
+    private Gtk.SizeGroup size_group = new Gtk.SizeGroup (Gtk.SizeGroupMode.HORIZONTAL);
```

Comment:
You could initialize this field in the `construct` block:
`size_group = new Gtk.SizeGroup (Gtk.SizeGroupMode.HORIZONTAL);`

Reply:
Done... do I need to make a new pull request or it will update the request automatically?